internal/pkg/service/templates/api/service/mapper.go

```diff
@@ -84,6 +84,19 @@ func (m Mapper) TaskPayload(model task.Task) (r *Task) {
 				InstanceID: &v,
 			}
 		}
+
+		if v, ok := model.Outputs["configId"].(float64); ok {
```

Comment:
why the float handling here?

Reply:
I'll check it, it is due to some internal parsing to `uint64`->`any`, it converts it back to `float64` for some unknown reason.

Reply:
https://keboola.atlassian.net/browse/PSGO-973